CHANGELOG.md

```diff
@@ -2,6 +2,68 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 52 end 2017-01-16
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+52+Ending+Jan+16%2C+2017%22+label%3Aenhancement)
+
+- Automate
+  - Move MiqAeEngine components into the appropriate directory ([#13406](https://github.com/ManageIQ/manageiq/pull/13406))
```

Comment:
Maybe include something like "in preparation to extract the Automate engine into a separate repo."